lib/lowkiq.rb

```diff
@@ -109,4 +109,5 @@ def build_by_node_splitter(number_of_nodes, node_number)
   self.dump_payload = ::Marshal.method :dump
   self.load_payload = ::Marshal.method :load
   self.workers = []
+  self.save_backtrace = false
```

Comment:
It will be better to use `self.format_error_message = :message.to_proc` and then `job[:error] = Lowkiq.format_error_message.call(ex)`
At least it requires no tests.
But the UI issues stay the same.